www/scipylib/building/windows.rst

```diff
@@ -5,374 +5,220 @@ Building From Source on Windows
-Fortran 77 interface).
-
-.. _BLAS: http://www.netlib.org/blas/faq.html
-.. _LAPACK: http://www.netlib.org/lapack/faq.html
-
-Pre-built versions of the ATLAS libraries are available for several 
-processors:
-
-.. TODO: Add links
-
- * Pentium 2; early Athlon chips
- * Pentium 3/SSE (possibly Athlon XP model 6 and later AMD chips)
- * Pentium 4/SSE2
-
-Building ATLAS
-##############
-
-ATLAS_ is the most widely available, free BLAS implementation on Windows. It 
-is well supported by NumPy and SciPy.
-
-**IMPORTANT:** NumPy and SciPy in Windows can currently only make use of CBLAS
-and LAPACK as *static libraries* - DLLs are not supported. 
-
-If you don't yet have optimized static CBLAS and LAPACK libraries, you can
-easily build them from within Cygwin (LAPACK also can just as easily be built
-with MinGW).
-
- 1. Download and extract the most recent version of the ATLAS sources.
-    Currently the most stable "unstable" version is 3.7.11. A new "stable"
-    version is expected to be released this summer, the 3.6.0 version is
-    already pretty dated. 
- 2. To avoid `SSE3 problems`_ on some platforms, deactivate SSE3 by replacing
-    line 77 in ``ATLAS/CONFIG/probe_SSE3.c`` with  
-
-    ::
-        
-        /* if (testv3[0] != 3.0 || testv3[1] != 7.0) */ 
-    
- 3. Execute make in the Cygwin command prompt in the Atlas root directory. In
-    Cygwin the Windows drives ``C:\``, ``D:\``, etc. are mapped to
-    ``/cygdrive/c/``, ``/cygdrive/d/``, etc. 
- 4. Generally accept the default options by hitting return. Select the correct
-    processor. Do not activate POSIX threads. Use the express installation.
-    You do not need to specify custom compiler flags, the ``-mno-cygwin`` does
-    not make a difference at this stage. Accept the architecture defaults. If
-    you do not know your processor type, downloading and running `CPU-Z`_ may
-    help.
- 5. As prompted by the config script, execute  ``make install
-    arch=YOUR_ARCHITECTURE`` . This can take anywhere from 15 minutes to
-    several hours, depending on your setup. 
- 6. Execute ``make sanity_test arch=YOUR_ARCHITECTURE``  and hope that no tests
-    fail (the message ``[sanity_test] Error 1 (ignored)`` is to be expected). 
-
-Now copy the files ``libatlas.a``, ``libcblas.a``, ``libf77blas.a`` and
-``liblapack.a`` from ``ATLAS\lib\YOUR_ARCHITECTURE`` to a directory of your
-choice, for example ``C:\BLASLAPACKLIBS``. 
-
-Building LAPACK
+Microsoft Visual C++ (MSVC)
+###########################
+
+NumPy and SciPy both support MSVC as the C/C++ compiler extension modules for the official
+binary distribution of Python. However, make sure that you download the correct version!
+For example, Python 2.7.x is compiled with Visual Studio 2008, and Python 3.5.1 is compiled
+with Visual Studio 2015. Once you have found the appropriate version, visit this link `here <https://support.microsoft.com/en-gb/kb/2977003>`__
```

Comment:
This links to the 'Visual C++ redistributable package', which is not enough to build yourself